### PR TITLE
Adding default tokens to ActionIcon

### PR DESCRIPTION
**Pull request by Jialecl**
**Checklist**
_(Check off all the items before submitting)_

- [x] Build process is done without errors. All tests pass in the `/lib` directory.
- [x] Self-reviewed the code before submitting.
- [x] Meets accessibility standards.
- [x] Added/updated documentation to `/website` as needed.
- [x] Added/updated tests as needed.

**Purpose**
Some default styles were added in case there are no tokens available.
This will allow the component to be used in isolation.

### CODE REVIEW

**Review by Mil4n0r — CHANGES_REQUESTED** [review]
img in line 21 of `ActionIcon.tsx` should have an `alt` text.

Overall, more extra coverage could be added to the testing, both in this Component and `Button`, but I think this exceeds the scope of this task, I would create a separate PR for that.

Some ideas could be:
- Ensure that `onClick` action is not triggered when the Component is disabled.
- Verify that the tabIndex is being correctly set and ensure that keyboard interaction is working as expected.
- Test whether the component renders correctly when the icon prop is of type string or JSX.

**Comment by Jialecl** [comment]
> * Verify that the tabIndex is being correctly set and ensure that keyboard interaction is working as expected.
The keyboard interaction is not needed to be tested, since it is the HTML behavior.

**Review by Mil4n0r — CHANGES_REQUESTED** [review]

File: lib/src/action-icon/ActionIcon.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import ActionIconProps, { RefType } from "./types";
 import styled from "styled-components";
+import CoreTokens from "../common/coreTokens";
 
 const DxcActionIcon = React.forwardRef<RefType, ActionIconProps>(
   ({ disabled = false, title, icon, onClick, tabIndex }, ref): JSX.Element => {
```
[the remark this answers is not in the document]
Comment (Jialecl) [reply lib/src/action-icon/ActionIcon.tsx:7]: The props were correctly typed.
You can hover each prop to see the types coming from "./types"

[the remark this answers is not in the document]
Comment (Mil4n0r) [reply lib/src/action-icon/ActionIcon.tsx:7]: You are right, it was a bug in my IDE.

**Review by Mil4n0r — APPROVED** [review]

**Pull request merged**